scripts/mbedtls_framework/ecp.py

```diff
@@ -164,7 +165,10 @@ def result(self) -> List[str]:
 
     @property
     def is_valid(self) -> bool:
-        return True
+        if build_tree.is_mbedtls_3_6():
+            return True
+        else:
+            return False
```

Comment:
That's a bit of a roundabout way to write `return build_tree.is_mbedtls_3_6()`. Not a blocker for me, but it is a blocker for pylint.